python/packages/nisar/products/writers/GcovWriter.py

```diff
@@ -608,7 +616,8 @@ def populate_processing_information(self):
             'algorithm_type']
         if geocoding_algorithm == 'area_projection':
             geocoding_algorithm_name = ('Area-Based SAR Geocoding with'
-                                        ' Adaptive Multilooking (GEO-AP)')
+                                        ' Adaptive Multilooking (GEO-AP).'
+                                        ' DOI: 10.1109/TGRS.2022.3147472')
```

Comment:
@gshiroma  Is CEOS asking for the DOI of the product or DOI of an algorithm that used in some modules to generate the product? Would you please copy the text of what they are asking for ? or point me to their doc?